client/src/legacy/entwine/LeftAndMain_Subsites.js

```diff
@@ -0,0 +1,190 @@
+/* jslint browser: true, nomen: true */
```

Comment:
seems like this file should have a .jsx extension

Reply:
This is not the Silverstripe way. All supported modules use .js for react components as well as other types of JS.
I actually agree, but consistency is a nice thing.